README.md

```diff
@@ -82,45 +91,56 @@ public class PushOneDocument {
 ```
 
 ## Logging
-When pushing multiple documents into your source using a service (e.g. `PushService`, `StreamService`), make sure to configure a **logger** to be able to see what happens.
-to do so .. in your `resources` folder.
 
-### Log4j2 XML Configuration Example
-To log execution output into the console, use the below `log4j2.xml` configuration:
-```xml
-<!-- log4j2.xml -->
-<?xml version="1.0" encoding="UTF-8"?>
-<Configuration>
-    <Appenders>
-        <Console name="ConsoleAppender" target="SYSTEM_OUT">
-            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" />
-        </Console>
-    </Appenders>
-    <Loggers>
-        <Root level="debug">
-            <AppenderRef ref="ConsoleAppender" />
-        </Root>
-    </Loggers>
-</Configuration>
-```
+If you want to push multiple documents to your Coveo organization and use a service for that (e.g. `PushService`, `StreamService`), you need to configure a **logger** to be able to see what happens.
```

Comment:
It is not required to configure a logger. The logger is useful to catch error messages/warnings from the SDK

Reply:
@y-lakhdar thx a lot! I've updated the wording here:

> If you want to push multiple documents to your Coveo organization and use a service for that (e.g. `PushService`, `StreamService`), you may find it useful to configure a **logger** to catch error and warning messages.